docs/resources/Guild.md

```diff
@@ -44,6 +44,8 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | vanity\_url\_code | ?string | the vanity url code for the guild |
 | description | ?string | the description for the guild |
 | banner | ?string | [banner hash](#DOCS_REFERENCE/image-formatting) |
+| premium_tier | integer | [premium tier](#DOCS_REFERENCE/guild-object-premium-tier)
+| premium_subscription_count? | integer | the total users subscribed to the guilds premium
```

Comment:
Let's also make this: `the total number of users who have boosted this server`

Reply:
I think "who are boosting" is more accurate than "who have boosted", since boosts are temporary & expire. The past tense might be confused with something like "total boosts to date"

Reply:
Something like "the total number of users currently boosting this server" would be more explicit in that case, and (I think) slightly more grammatically correct.

Reply:
I'm good with Geo's suggestion 👍